bpf/bpf_host.c

```diff
@@ -1324,6 +1326,18 @@ int cil_from_netdev(struct __ctx_buff *ctx)
 __section("from-host")
 int cil_from_host(struct __ctx_buff *ctx)
 {
+#ifdef ENABLE_HIGH_SCALE_IPCACHE
+	int ret = CTX_ACT_OK;
```

Comment:
Is this needed for a later patch? Right now it's a bit confusing (given we never use the value)